Formula/m/micropython.rb

```diff
@@ -16,7 +16,7 @@ class Micropython < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "python@3.11" # Requires python3 executable
+  depends_on "python@3.12" # Requires python3 executable
```

Comment:
is the comment stale? may also want to check if this can just be `uses_from_macos`?

Reply:
yeah, I guess so, let me try that.